### ISSUE TITLE
No checkra1n app on iPad Pro 10.5 13.2.2 

### ISSUE THREAD

**Issue by XanDesign** [checkra1n/BugTracker#42]
**What device + iOS version are you on?**
iPad Pro 10.5 13.2.2 64GB Cellular

**What are the steps to reproduce the issue?**
1. open checkra1n on mac
2. enter dfu on device
3. jailbreak 
...

**What do you expect, and what is happening instead?**
There is no checkra1n boot screen during the "booting" segment. Just a plain old apple. When it boots, checkra1n still says "booting" even after it has booted. There is no checkra1n app on ipad homescreen and no way to move forward from there. 

**Any other info, error logs, screenshots, ...?**

**Comment by cjf99** [comment]
same here iPhone 10,6 ios 13.0. after a while i get the error -20

**Comment by FaZeBase** [comment]
Have you tried closing checkra1n, putting the device in DFU mode and starting over ?

**Comment by cjf99** [comment]
yes I've tried it like 10+ times now. sometimes there is a checkra1n logo but it only says caught fault twice and then a bunch of numbers and letters (hexadecimals i guess) and sometimes it just boots normally like he said

**Comment by fuomag9** [comment]
Ipad pro 10.5 first gen is unsupported

**Comment by dylanneve1** [comment]
Go to applications go tocheckra1n and then into Contents/MacOS. Open a terminal here and run "./checkra1n_gui". It will open up a new window of checkra1n but you will also have a console logging whats going on. 

Run the program normally and when -20 error shows up just leave it running. In the console you should see the error and a line beneath saying just ">>>>". Leave the device and don't touch it. Eventually you will get a message saying packet sent or something similar and the app should appear. This worked for me. Let me know what you find.

**Comment by cjf99** [comment]
i cant find contents/macos

**Comment by dylanneve1** [comment]
@cjf99 open a terminal window and CD here "/Applications/checkra1n.app/Contents/MacOS"

**Comment by cjf99** [comment]
yeah already found it thanks. I'll just let it sit then. did it show any kind of checkra1n logo or something while booting up?

**Comment by cjf99** [comment]
Last login: Sun Nov 10 23:29:00 on ttys000
Colins-MacBook-Pro:~ colin$ /Applications/checkra1n.app/Contents/MacOS/checkra1n_gui ; exit;
- [*] Waiting for DFU devices
- [*] DFU mode device found
- [*] Attempting to perform checkm8 on 8015 11...
- [*] == Checkm8 Preparation stage ==
- [*] Stalled input endpoint
- [*] DFU mode device found
- [*] == Checkm8 Setup stage ==
- [*] Entered initial checkm8 state after 3 steps, issuing DFU abort..
- [*] DFU device disconnected
- [*] DFU mode device found
- [*] == Checkm8 Trigger stage ==
- [*] Checkmate!
- [*] DFU device disconnected
- [*] DFU mode device found
- [*] == Checkm8 Trying to run payload... ==
- [*] If everything went correctly, you should now have code execution.
- [*] DFU device disconnected
- [*] Download mode device found
- [*] Download mode device disconnected
- [!] Timed out waiting for bootstrap upload (Error -20)
thats what i get. no >>>> just an error message that asks me if I want to send details about the error

**Comment by dylanneve1** [comment]
@cjf99 hmm leave it on this screen for a couple minutes and see. I did get the checkra1n logo while booting are you getting it?

**Comment by cjf99** [comment]
no i just get the normal apple logo... as is said before sometimes im getting some weird checkra1n screen but that only happened like twice

**Comment by dylanneve1** [comment]
What is the weird screen? The one they posted on twitter with the text saying proudly written in nano etc etc or something different?

**Comment by cjf99** [comment]
i get exactly the same screen as the guy in #14 it just says caught fault twice and then some numbers

**Comment by cjf99** [comment]
I got it fixed! Just do everything in the program but when the dfu guide comes start at 5 seconds not at 6 at the first step. Everything else normally! Try it a few times for me it worked on the 2nd try. Shoutout to @opa334 for telling me on twitter!
